test/e2e/framework/framework.go

```diff
@@ -0,0 +1,44 @@
+package framework
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var Global *Framework
+
+type Framework struct {
+	KubeConfig *rest.Config
+	KubeClient kubernetes.Interface
+}
+
+func setup() error {
+	defaultKubeConfig := ""
+	homedir, ok := os.LookupEnv("HOME")
+	if ok {
+		defaultKubeConfig = homedir + "/.kube/config"
+	}
+	config := flag.String("kubeconfig", defaultKubeConfig, "kube config path, e.g. $HOME/.kube/config")
```

Comment:
`kube config path, e.g. $HOME/.kube/config` ==> `kubeconfig path, defaults to $HOME/.kube/config`